libr/util/d/pokered

```diff
@@ -0,0 +1,63 @@
+0x7f=-
```

Comment:
rename to pokered.sdb.txt and compile them with r2sdb like its done in other modules

Reply:
@trufae I did not find the executable r2sdb. I just found sdb instead. Where can I find it?

Reply:
I renamed it but I can not compile it.